backend/inspirehep/matcher/config.py

```diff
@@ -271,3 +271,20 @@
     "index": "records-data",
     "source": ["control_number"],
 }
+
+REFERENCE_MATCHER_AUTHORS_CONFIG = {
+    "algorithm": [
+        {
+            "queries": [
+                {"path": "ids.value", "search_path": "ids.value", "type": "exact"},
+                {
+                    "path": "emails",
+                    "search_path": "email_addresses.value",
+                    "type": "exact",
+                },
+            ]
+        }
+    ],
+    "index": "records-authors",
+    "source": ["control_number"],
```

Comment:
you should also add a validator, to make sure you're not matching an ID with one `schema` with an ID with a different `schema`, same idea as in https://github.com/inspirehep/inspire-matcher/blob/0ca79e7d758dea2f6ee273dcbf7b9fa15f3c4dbb/inspire_matcher/validators.py#L78

Reply:
Wouldn't it be too expensive? We'll have to query pidstore every time to check the record type

Reply:
no, it's part of the record (`ids` contains `schema` and `value`).